### PR TITLE
[@types/react-table] Property does not exist on type 'object'.

### PR DESCRIPTION
**Pull request by Havret**
#22947

### CODE REVIEW

**Comment by Havret** [comment]
Not sure why the build failed. There is no log available.

File: types/react-table/react-table-tests.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const columns: Column[] = [
   {
     Header: 'Stats',
     columns: [
-      { Header: "Visits", accessor: "visits" }
+      { Header: "Visits", accessor: (row: any) => row.visits }
```
Comment (andy-ms) [inline types/react-table/react-table-tests.tsx]: This test would actually work even without the change, due to the explicit type annotation of `: any` being compatible with anything.

File: types/react-table/index.d.ts [hunk 2]
```diff
@@ -1,12 +1,12 @@
-// Type definitions for react-table 6.7
+// Type definitions for react-table 6.8
```
Comment (andy-ms) [inline types/react-table/index.d.ts]: Don't bump this unless the library version changed.

**Review by ghost — CHANGES_REQUESTED** [review]

File: types/react-table/index.d.ts (hunk 2 above)
Comment (Havret) [reply types/react-table/index.d.ts]: Yep, I just want to trigger build on CI.

File: types/react-table/react-table-tests.tsx [hunk 3]
```diff
@@ -23,7 +23,7 @@ const columns: Column[] = [
   {
     Header: 'Stats',
     columns: [
-      { Header: "Visits", accessor: "visits" }
+      { Header: "Visits", accessor: row: => row.visits }
```
Comment (andy-ms) [inline types/react-table/react-table-tests.tsx]: Remove the `:`

**Comment by Havret** [comment]
@andy-ms It's running under noImplicitAny flag, thus I has to be done that way:

`      { Header: "Visits", accessor: (row: any) => row.visits }
`

**Comment by andy-ms** [comment]
It looks like that's because `interface Column` declares `columns?: any[]`, so the entire object there is typed as `any`. Should it be `columns?: Column[]`?

**Comment by Havret** [comment]
@andy-ms It's running under noImplicitAny flag, thus I has to be done that way:

{ Header: "Visits", accessor: (row: any) => row.visits }

**Comment by andy-ms** [comment]
@Havret the reason that is implicitly-any is that the whole object there is typed as `any`, because it's in the value of `colums` which is currently typed as `any[]`. Could it be typed as `Column[]` instead?

**Comment by mhegazy** [comment]
Thanks for your contribution. This PR has failing CI tests and can not be merged in at the time being. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

**Pull request closed without merging**